src/agent/solana.rs

```diff
@@ -75,14 +75,19 @@ pub mod network {
         keypair_request_tx: mpsc::Sender<KeypairRequest>,
         logger: Logger,
     ) -> Result<Vec<JoinHandle<()>>> {
+        // Permissioned price account updates between oracle and exporter
+        let (perm_price_tx, perm_price_rx) = mpsc::channel(config.oracle.updates_channel_capacity);
```

Comment:
just to confirm, is this config ok here?

Reply:
This value felt intuitive, because the permissioned accounts also represent updates coming from the oracle